tvtk/pyface/ui/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -424,11 +432,37 @@ def keyReleaseEvent(self, ev):
         self._Iren.KeyReleaseEvent()
 
     def wheelEvent(self, ev):
+        """ Reimplemented to work around scrolling bug in Mac.
+
+        Work around https://bugreports.qt-project.org/browse/QTBUG-22269.
+        Accumulate wheel events that are within a period of 25ms into a single
+        event.  Changes in buttons or modifiers, while a scroll is going on,
+        are not handled, since they seem to be too much of a corner case to be
+        worth handling.
+        """
+        self.wheel_accumulator += ev.delta()
+        self._saved_wheel_event_info = (
```

Comment:
Well Ctrl+Scroll or etc have different in many apps. If VTK does not provide a way to handle them then i guess it is not needed, but in the general case it is.
For example, in browsers, Cmd+scroll usually zooms, so there would be upto 25ms error in handling switching. Maybe it does not matter